R/pkg/R/DataFrame.R

```diff
@@ -2880,7 +2880,7 @@ setMethod("fillna",
 #'
 #' @param x a SparkDataFrame.
 #' @param row.names NULL or a character vector giving the row names for the data frame.
```

Comment:
I'm not sure if we want to use \code{NULL} or not, considering we use \code{TRUE} for logical values.